lib/tasks/backfill_daily_stats_analytics.rake

```diff
@@ -0,0 +1,7 @@
+desc 'For backfilling analytics CSVs'
+task backfill_daily_stats: :environment do
+  (Date.new(2021,9,6)..Date.new(2021,9,14)).each do |date|
+    SendAnalytics::DailyStatsJob.perform_later(date: date)
+    SendAnalytics::DecisionsJob.perform_later(date: date)
+  end
+end
```

Comment:
Have tested with my GCS instance and this correctly creates the CSVs:

<img width="591" alt="Screenshot 2021-09-15 at 16 17 08" src="https://user-images.githubusercontent.com/6339025/133461541-3885af41-aafd-48ba-a18e-e807632c53d1.png">
<img width="583" alt="Screenshot 2021-09-15 at 16 16 59" src="https://user-images.githubusercontent.com/6339025/133461551-0d4a35dd-3038-4c71-92a7-79fa1e70bf05.png">

Reply:
@d-a-v-e Great documentation, really appreciate this style. It resonates with how I like to work and make PR's easily understandable :+1: